inception/inception/slim/ops_test.py

```diff
@@ -602,7 +600,8 @@ def testComputeMovingVars(self):
       update_ops = tf.get_collection(ops.UPDATE_OPS_COLLECTION)
       with tf.control_dependencies(update_ops):
         barrier = tf.no_op(name='gradient_barrier')
-        output = control_flow_ops.with_dependencies([barrier], output)
+        with tf.control_dependencies([barrier]):
+          output = tf.identity(output)
```

Comment:
You mean moving `ops.batch_norm(images, decay=0.1)` to this line, right? Not quite sure why, but that causes the tests to fail (something to do with the `update_ops` above).